Hannes7eicher/Plant-Watering-Kit-Asset-Update [MKC-828] (#828)
arduino/docs-content: content/hardware/08.kits/maker/plant-watering-kit/tutorials/iot-smart-garden-setup-with-nanorp2040/content.md

![GIF of zip-tying the cables](assets/ziptie.gif)

## IoT Cloud Setup
If you are new to the Arduino IoT Cloud, check out our [Getting Started Guide](https://docs.arduino.cc/arduino-cloud/getting-started/iot-cloud-getting-started).

### Template

> We also provide a template which you can use to get started without the need to configure the entire IoT Cloud setup manually. Check out [IoT Cloud Template](https://create.arduino.cc/iot/templates/plant-watering-kit) for an automated setup process.

Reviewer: ```suggestion
To connect your board to the Arduino IoT Cloud, we will use the [Plant Watering Kit Template](IoT Cloud Template](https://create.arduino.cc/iot/templates/plant-watering-kit). This template installs a specific sketch on your board and creates a dashboard that allows you to interact with your board: you don't need to write any code at all!

See the image below to understand how to set it up.
```

Added an explanation to what a template is

![Thing overview complete.](assets/template_overview.png)

> ### Setup

Reviewer: ```suggestion
### Manual Setup
```

> 

Reviewer: ```suggestion
***This section is for you who want to set up the cloud manually. For easy setup, use the template provided (see section just above).*** 
```
